pkg/tests/api/stats/admin_test.go

```diff
@@ -0,0 +1,92 @@
+package stats
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/services/org"
+	"github.com/grafana/grafana/pkg/services/org/orgimpl"
+	"github.com/grafana/grafana/pkg/services/quota/quotaimpl"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/supportbundles/supportbundlestest"
+	"github.com/grafana/grafana/pkg/services/user"
+	"github.com/grafana/grafana/pkg/services/user/userimpl"
+	"github.com/grafana/grafana/pkg/tests/testinfra"
+)
+
+func TestAdminStats(t *testing.T) {
```

Comment:
```suggestion
func TestIntegrationAdminStats(t *testing.T) {
```
see https://github.com/grafana/grafana/pull/59521